cirq-google/cirq_google/engine/engine.py

```diff
@@ -800,8 +811,8 @@ def get_engine(project_id: Optional[str] = None) -> Engine:
         The Engine instance.
 
     Raises:
-        EnvironmentError: If the environment variable GOOGLE_CLOUD_PROJECT is
-            not set.
+        OSError: If the environment variable GOOGLE_CLOUD_PROJECT is not set. This is actually
+            an `EnvironmentError`, which by definition is an `OsError`.
```

Comment:
Technically correct, the best kind of correct.

Reply:
Lint yells if you call it an `EnvironmentError`, so this is the annoying workaround.